src/renderer/widgets/Dropdown.tsx

```diff
@@ -136,26 +118,14 @@ const Dropdown = ({
         <ArrowIcon type="down" className="arrow" />
       </div>
       {isOpen && (
-        <ul aria-label={`${name}-items`}>
-          {options.map((entry: IOptionEntry, index: number) => {
-            return (
-              <li
-                role="menuitem"
-                ref={inputRefs[index]}
-                className="row"
-                key={entry.value}
-                value={entry.value}
-                aria-label={entry.label}
-                onClick={() => onChange(entry.value)}
-                onKeyDown={(e) => handleItemKeyPress(e, entry, index)}
-                onMouseEnter={() => inputRefs[index].current?.focus()}
-                tabIndex={0}
-              >
-                {entry.display}
-              </li>
-            );
-          })}
-        </ul>
+        <List
```

Comment:
Since inputRefs is no longer set here, the useEffect of line 53 is pointless right?